docs/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+#### What does this PR do?
+
+#### Where should the reviewer start?
+
+#### Ideas that were considered and discarded
+
+#### Relevant tickets
```

Comment:
Definitely needed. I often forget this bit.

Reply:
Definitely add the `closes [ticketnumbers]` part! I forget that too often.